ElasticsearchReportHelper.php

```diff
@@ -657,7 +658,32 @@ public static function permissionFilters(array $options) {
       'class' => 'permissions-filter',
     ];
 
-    return data_entry_helper::select($controlOptions);
+    $dropdown = data_entry_helper::select($controlOptions);
+    $html = <<<HTML
+<div>
+  $dropdown 
+  <div id="permission-filters-notice"></div>
+</div>
+
+HTML;
+
+  helper_base::$indiciaData['esPermissionFilterNotices'] = json_encode($options['notices']);
+  helper_base::$late_javascript .= <<<JS
```

Comment:
Although our old code is littered with inline JS, I would like to avoid it where possible for new code. Each control has a JS file in media/js/indicia.datacomponents called jquery.idc.*.js where * is the control name. These are plugins with a defined structure to copy from.

Reply:
Removed inline JS to plugin and used data attribute to store notices.